pipelines/rj_smas__disparo_template/flow.py

```
@@ -0,0 +1,192 @@
+    id_hsm: int | None = None,
+    campaign_name: str | None = None,
+    cost_center_id: int | None = None,
+    chunk_size: int | None = None,
+    dataset_id: str | None = None,
+    table_id: str | None = None,
+    dump_mode: str | None = None,
+    test_mode: bool | None = True,
+    query: str | None = None,
+    query_processor_name: str | None = "template",
+    sleep_minutes: int | None = 5,
+    infisical_secret_path: str = "/wetalkie",
```

Ruff (0.14.3)

Possible hardcoded password assigned to function default: "infisical_secret_path"

(S107)